src/pudl/metadata/enums.py

```diff
@@ -213,3 +213,35 @@
     "Unknown Code",  # Should be replaced with NA
 ]
 """Valid emissions measurement codes for the EPA CEMS hourly data."""
+
+TECH_DESCRIPTIONS: list[str] = [
```

Comment:
I tried converting all the `list` enums to `set` which broke a bunch of stuff because some of the enums are used in for concatenation with a list. Just updated the enums I added (`TECH_DESCRIPTIONS` and `PLANT_PARTS`) to be sets.